playbooks/roles/go-server/tasks/download_backup.yml

```
@@ -59,13 +59,14 @@
     expiration=30
   when: GO_SERVER_BACKUP_S3_BUCKET and GO_SERVER_BACKUP_S3_OBJECT
   register: s3_one_time_url
+  ignore_errors: yes
```

Why ignore errors on this?